hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/contracts/operation/HederaCreate2Operation.java

```diff
@@ -16,24 +16,15 @@
 
 package com.hedera.node.app.service.mono.contracts.operation;
 
-import static com.hedera.node.app.service.mono.sigs.utils.MiscCryptoUtils.keccak256DigestOf;
-import static org.hyperledger.besu.evm.internal.Words.clampedToLong;
-
+import com.hedera.node.app.service.evm.contracts.operations.HederaEvmCreate2Operation;
 import com.hedera.node.app.service.mono.context.properties.GlobalDynamicProperties;
 import com.hedera.node.app.service.mono.records.RecordsHistorian;
 import com.hedera.node.app.service.mono.state.EntityCreator;
-import com.hedera.node.app.service.mono.store.contracts.HederaStackedWorldStateUpdater;
 import com.hedera.node.app.service.mono.store.contracts.precompile.SyntheticTxnFactory;
 import javax.inject.Inject;
-import org.apache.tuweni.bytes.Bytes;
-import org.apache.tuweni.bytes.Bytes32;
-import org.apache.tuweni.units.bigints.UInt256;
-import org.hyperledger.besu.datatypes.Address;
-import org.hyperledger.besu.evm.frame.MessageFrame;
 import org.hyperledger.besu.evm.gascalculator.GasCalculator;
 
-public class HederaCreate2Operation extends AbstractRecordingCreateOperation {
-    private static final Bytes PREFIX = Bytes.fromHexString("0xFF");
+public class HederaCreate2Operation extends HederaEvmCreate2Operation {
```

Comment:
If the only need for having a specific services version of this Op Code is to initialize the `HederaCreateOperationExternalizer` we can do this initialization via injecting it as a Dagger dependency and register `HederaEvmCreate2Operation` instead of `HederaCreate2Operation`(in `ContractsV_0_30Module` and `ContractsV_0_34Module`). Then we can delete the whole class.